### DISCUSSION TITLE
Enhancement: Allow Manual Regeneration of Conversation Titles

### DISCUSSION THREAD

**Discussion by RedwindA** [danny-avila/LibreChat#2890] [Feature Requests & Suggestions]
What features would you like to see added?
I would like to see a feature that allows users to manually regenerate the conversation title after the initial LLM-generated title.

More details
Sometimes, due to various reasons such as temporary API issues or an abnormal termination of the first round of conversation, LibreChat fails to generate a title. Additionally, as the conversation progresses, the initial title may not accurately summarize or describe the entire conversation. Allowing users to manually regenerate the title would be beneficial in these cases.

And I hope to allow the use of the full text for title generation, providing an option for those who are not concerned about the cost

**Comment by RedwindA** [comment]
any interest?

**Reply by danny-avila** [reply]
It should be relatively simple, @berry-13 was considering this the other day.

**Reply by RedwindA** [reply]
@berry-13

**Comment by Physicsly** [comment]
Any update on this?
